### PR TITLE
Upgrade everything

### PR DESCRIPTION
**Pull request by jonmattgray**
Removes [`chai-json` ](https://www.npmjs.com/package/chai-json) - a barely used and unmaintained package

### CODE REVIEW

File: test/integration/docs.test.js [hunk 1]
```diff
@@ -17,7 +16,6 @@ describe('async-docs', function () {
   })
 
   it('successfully returns the asyncapi docs json', function () {
-    expect(context.response.body).to.be.a.jsonObj()
```
Comment (jonmattgray) [inline test/integration/docs.test.js:20]: the only place where `chai-json` was used

Comment (mattdean-digicatapult) [reply test/integration/docs.test.js:20]: can we have an alternate assertion?

Comment (jonmattgray) [reply test/integration/docs.test.js:20]: how about https://github.com/digicatapult/wasp-ws-reading-service/pull/2/commits/9fc8844a51b237c7a69df08d3be35b7fc51d56e1?

**Review by mattdean-digicatapult — APPROVED** [review]

File: test/integration/docs.test.js (hunk 1 above)
Comment (mattdean-digicatapult) [reply test/integration/docs.test.js:20]: ugly but it will do

**Pull request merged**